src/commands/module/remove.ts

```diff
@@ -115,6 +115,8 @@ export default class Remove extends BaseCommand<typeof Remove> {
       serviceConfig,
     );
 
+    // Our configs can't be Map object so there is no workaround for this rule
+    // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
     delete serviceConfig.modules[moduleNameToRemove];
```

Comment:
I agree it can be dealt with. Even created a task https://linear.app/fluence/issue/DXJ-484/use-js-map-objects-instead-of-plain-js-object-for-dynamic-hashmaps-in
This would very rarelly cause a problem but it would require time to fix this, so if you don't mind I will do it later in a separate PR